packages/subgraph/src/mappingHelpers.ts

```diff
@@ -152,20 +161,25 @@ export function getOrInitSuperToken(
  */
 export function getOrInitToken(
     tokenAddress: Address,
-    block: ethereum.Block
+    block: ethereum.Block,
+    resolverAddress: Address
 ): void {
-    let tokenId = tokenAddress.toHex();
+    const tokenId = tokenAddress.toHex();
     let token = new Token(tokenId);
 
     if (tokenAddress.equals(ZERO_ADDRESS)) {
         return;
     }
 
     token.createdAtTimestamp = block.timestamp;
+    token.decimals = 0;
+    token.name = "";
+    token.symbol = "";
     token.createdAtBlockNumber = block.number;
     token.isSuperToken = false;
+    token.isNativeAssetSuperToken = false;
```

Comment:
logic change alert: isNativeAssetSuperToken set to false for underlying token

Reply:
When I check existing Subgraphs then it is `false` there too. This explicitness is forced by Matchstick and it shouldn't have effect on production?

Reply:
yup, same thing as above where it expects not null, but is null. it has no effect on production as all underlying are false currently and if not, should be.